FIRO_TSEnsembles/src/main/java/hec/JdbcDatabase.java

```diff
@@ -18,11 +18,12 @@
 
 import hec.ensemble.*;
 import hec.paireddata.*;
+import hec.metrics.*;
 
 /**
  * Read/write Ensembles to a JDBC database
  */
-public class JdbcTimeSeriesDatabase extends TimeSeriesDatabase {
+public class JdbcDatabase implements PairedDataDatabase, EnsembleDatabase, VersionableDatabase, MetricDatabase {
```

Comment:
thinking about other names for JdbcDatabase:   HecDatabase,  StudyDatabase, DssDatabase...

Reply:
we are writing to sqlite... maybe SqliteDatabase

Reply:
SqliteDatabase is the best yet.